clients/web/src/lib/components/abstract/Chatwoot/Chatwoot.svelte

```diff
@@ -0,0 +1,45 @@
+<script lang="ts">
+    import {onMount} from "svelte";
+    import {browser} from "$app/env";
+    import {session} from "$app/stores";
+
+    import {chatwootSettings} from "./Chatwoot.constants";
+
+    const {
+        enabled, url, websiteToken,
+    } = $session.config.chatwoot;
+
+    onMount(() => {
+        if (browser && enabled) {
+            (function() {
+                const g = document.createElement("script");
+                g.src = `${url}/packs/js/sdk.js`;
+                g.defer = true;
+                g.async = true;
+                document.body.append(g);
+                g.onload = function() {
+                    window.chatwootSettings = chatwootSettings;
+                    window.chatwootSDK?.run({
+                        websiteToken: websiteToken,
+                        baseUrl: url,
+                    });
+                };
+            })();
+        }
+    });
+
+    if (browser) {
+        window.addEventListener("chatwoot:ready", () => {
+            if (!$session.user || !window.$chatwoot) return;
+
+            window.$chatwoot.setUser($session.user.userId.toString(), {
+                name: $session.user.username,
+            });
+            // TODO support other orgs in the future
+            window.$chatwoot.setLabel("org-mle");
```

Comment:
```suggestion
            window.$chatwoot.setLabel(`Organization Id: 2`);
```

Reply:
We should have tags that are more friendly to our support staff. In the client we can convert the org id to a readable tag like `org-mle`.